spec/lists_spec.rb

```diff
@@ -33,7 +33,7 @@
           headers: TOKEN_AUTH_HEADERS
         )
         .to_return(body: "{\"status\": \"ok\"}", status: 200)
-      res = client.lists.send(NOTIFICATION_ID, list: LIST_ID, brand: BRAND_ID, override: {"provider": {}})
+      res = client.lists.send(NOTIFICATION_ID, list_id: LIST_ID, brand: BRAND_ID, override: {"provider": {}})
```

Comment:
Just noticed this. We should only use `list_id` if it matches the API. Here it should be list `list`. There are some places in the Lists API where it is `list_id`. It just gets confusing. We have some issues with it in the node library.